tpcc/check.go

```diff
@@ -227,28 +231,33 @@ EXCEPT ALL
 
 func checkConsistency(db *sql.DB) bool {
 	type check struct {
-		name      string
-		f         func(db *sql.DB) error
-		expensive bool
+		name string
+		f    func(db *sql.DB) error
 	}
 
 	checks := []check{
-		{"3.3.2.1", check3321, false},
-		{"3.3.2.2", check3322, false},
-		{"3.3.2.3", check3323, false},
-		{"3.3.2.4", check3324, false},
-		{"3.3.2.5", check3325, false},
-		{"3.3.2.6", check3326, true},
+		{"3.3.2.1", check3321},
+		{"3.3.2.2", check3322},
+		{"3.3.2.3", check3323},
+		{"3.3.2.4", check3324},
+		{"3.3.2.5", check3325},
+		{"3.3.2.6", check3326},
 	}
 	var errorEncountered bool
 
 	// TODO(arjun): We should run each test in a single transaction as currently
 	// we have to shut down load before running the checks.
 
-	for _, check := range checks {
-		if check.expensive && !*expensive {
+	toRun := strings.Split(*checksToRun, ",")
+	for _, checkToRun := range toRun {
+		i, err := strconv.Atoi(checkToRun)
+		if err != nil {
+			fmt.Printf("error parsing check %s, skipping: %v", checkToRun, err)
+			errorEncountered = true
 			continue
 		}
+
+		check := checks[i-1]
```

Comment:
Not a big deal because this is run manually, but this will panic if you specify a check that doesn't exist. I suggest creating a map from int check id to check before this loop by looping through the checks, and then using the map in this loop instead of the slice. Then you can throw an error if the check is not found.